prep-html-links/contact-us/contact-us.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Contact Us</title>
+</head>
+<body>
+  <h1>Contact Us</h1>
+  <a href="file:///Users/kaiseki/repos/prep-0621-code-solutions/prep-html-links/index.html">Click here to go to the home page</a>
```

Comment:
Here you'll want to use the `..` syntax for making a relative URL in your `href` attribute.

```html
href="../index.html"
```

Otherwise, this link won't work properly unless the person opening it has the exact same file structure that you do.